smac/runhistory/runhistory.py

```diff
@@ -45,6 +46,7 @@ def __init__(self, aggregate_func):
         self._n_id = 0
 
         self.cost_per_config = {} # config_id -> cost
+        self.runs_per_config = {} # config_id -> number of runs
```

Comment:
Could you please add a comment that this is necessary to calculate the moving average?